extensions-contrib/azure-extensions/src/main/java/org/apache/druid/storage/azure/AzureDataSegmentPusher.java

```diff
@@ -157,6 +128,7 @@ public DataSegment push(final File indexFilesDir, final DataSegment segment, fin
       );
     }
     catch (Exception e) {
+      // do we want to throw a specfic exception here? Interface only expects IOExceptions to be thrown.
```

Comment:
Haha, is this question for the reviewers? I think `RuntimeException` is enough for now.

Reply:
removed comment